packages/react-next/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -425,6 +425,7 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
     // Popup uses to determine if focus is contained when dismissal occurs
     this._tryFocusPreviousActiveElement({
       containsFocus: this._focusingPreviousElement,
+      documentContainsFocus: this._targetWindow.document.hasFocus(),
```

Comment:
I don't believe you need to port this to the react-next package yet.

Reply:
Should I drop all changes in react-next, or just this api change